consul-secure-on-kubernetes/step1.md

```diff
@@ -0,0 +1,59 @@
+<style>
```

Comment:
It worries me to include inline, hard-coded CSS that is going to be difficult to maintain.

Is this important enough that we really need it, or can we use the built-in Katacoda styles?

Reply:
I did try to steal some of their css classes to varying degrees of success. I'll look for documentation on how to style the katacoda way.

Reply:
I guess I meant that we should do a regular code block and not try to style it at all (but you may be trying to achieve something different).

Reply:
Reverted to katacoda plaintext.